pkg/config/remote/service/service.go

```diff
@@ -83,6 +90,13 @@ type Service struct {
 	// via logs.
 	rcType string
 
+	uptane uptaneClient
+	db     *bbolt.DB
+}
```

Comment:
Go Style: Move the methods specific to the Service struct next to the struct definition. Not directly part of the PR I think because of the rename, so I couldn't actually put a comment next to the methods (lines 786 though IIRC).